tensorflow/lite/micro/tools/make/targets/bluepill_makefile.inc

```diff
@@ -74,8 +74,14 @@ MICROLITE_TEST_SRCS := $(filter-out $(EXCLUDED_TESTS), $(MICROLITE_TEST_SRCS))
 
 EXCLUDED_EXAMPLE_TESTS := \
   tensorflow/lite/micro/examples/magic_wand/Makefile.inc \
-  tensorflow/lite/micro/examples/micro_speech/Makefile.inc \
-  $(shell find google/ -name Makefile_internal.inc)
+  tensorflow/lite/micro/examples/micro_speech/Makefile.inc
+
+ifneq ($(EXTERNAL_DIR),)
+  EXCLUDED_EXAMPLE_TESTS += $(shell find $(EXTERNAL_DIR) -name Makefile_internal.inc)
+else
+  EXCLUDED_EXAMPLE_TESTS += $(shell find ../google/ -name Makefile_internal.inc)
+endif
```

Comment:
let's add a TODO here as well to remove google in the future.

And let's also make a new bug for why we are not running all the tests on bluepill -- that seems like something we should fix.

Reply:
For the first part, using b/246833632. For the second part, creating a placeholder bug and can add more update later.